OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json

```diff
@@ -22,346 +22,325 @@
     },
     {
       "InputDtmi": "dtmi:mapped:core:AHU;1",
-      "OutputDtmi": "dtmi:com:willowinc:AHU;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirHandlingUnit;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirAlarm;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Alarm;1"
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirDifferentialPressureSensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Differential_Pressure_Sensor;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirDeltaPressureSensor;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirDifferentialPressureSetpoint;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Differential_Pressure_Setpoint;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirDeltaPressureSetpoint;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirDiffuser;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Diffuser;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirDiffuser;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirEnthalpySensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Enthalpy_Sensor;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirEnthalpySensor;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowAlarm;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Alarm;1"
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowAlarmSetpoint;1",
-      // Invalid Output DTMI - Post Brick 1.3
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Alarm_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSetpoint;1",
       "IsIgnored": true
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowDeadbandSetpoint;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Deadband_Setpoint;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirFlowDeadbandSetpoint;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowDemandSetpoint;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Demand_Setpoint;1"
+      "OutputDtmi": "dtmi:com:willowinc:AirFlowSetpoint;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowDirectionStatus;1",
-      // Mapped to Base - Review needed
-      "OutputDtmi": "dtmi:com:willowinc:Status;1"
+      "OutputDtmi": "dtmi:com:willowinc:State;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowLossAlarm;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Loss_Alarm;1"
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowRateCommand;1",
-      // Invalid Output DTMI - Review Needed
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Rate_Command;1",
-      "IsIgnored": true
+      "OutputDtmi": "dtmi:com:willowinc:Actuator;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowSensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Sensor;1"
+      "OutputDtmi": "tmi:com:willowinc:AirFlowSensor;1"
```

Comment:
typo: missing the d in dtmi. Caused test to fail

Reply:
Done